flink-connector-mysql-cdc/src/test/java/com/alibaba/ververica/cdc/connectors/mysql/table/MySQLConnectorITCase.java

```diff
@@ -152,6 +152,67 @@ public void testConsumingAllEvents() throws SQLException, ExecutionException, In
 		result.getJobClient().get().cancel().get();
 	}
 
+	@Test
+	public void testIgnoreUnchangeUpdates()throws Throwable {
+		inventoryDatabase.createAndInitialize();
+		String sourceDDL = String.format(
+				"CREATE TABLE debezium_source (" +
+						" first_name STRING," +
+						" last_name STRING" +
+						") WITH (" +
+						" 'connector' = 'mysql-cdc'," +
+						" 'hostname' = '%s'," +
+						" 'port' = '%s'," +
+						" 'username' = '%s'," +
+						" 'password' = '%s'," +
+						" 'database-name' = '%s'," +
+						" 'table-name' = '%s'," +
+						" 'capture-unchanged-updates' = 'false'" +
+						")",
+				MYSQL_CONTAINER.getHost(),
+				MYSQL_CONTAINER.getDatabasePort(),
+				inventoryDatabase.getUsername(),
+				inventoryDatabase.getPassword(),
+				inventoryDatabase.getDatabaseName(),
+				"customers");
+		String sinkDDL = "CREATE TABLE sink (" +
+				" first_name STRING," +
+				" last_name STRING" +
+				") WITH (" +
+				" 'connector' = 'values'," +
+				" 'sink-insert-only' = 'false'" +
+				")";
+		tEnv.executeSql(sourceDDL);
+		tEnv.executeSql(sinkDDL);
+
+		// async submit job
+		TableResult result = tEnv.executeSql("INSERT INTO sink SELECT first_name, last_name FROM debezium_source");
+
+		waitForSnapshotStarted("sink");
+
+		try (Connection connection = inventoryDatabase.getJdbcConnection(); Statement statement = connection.createStatement()) {
+
+			statement.execute("UPDATE customers SET first_name='Isaiah' WHERE id=1001;");
+			statement.execute("UPDATE customers SET email='isaiah.thomas@acme.com' WHERE id=1001;");
```

Comment:
Put this statement before the first_name update. Because the sink may stop when process the first_name update (hit 6 records).